boa_engine/src/vm/flowgraph/mod.rs

```diff
@@ -45,7 +45,7 @@ impl CodeBlock {
 
             pc += size_of::<Opcode>();
             match opcode {
-                Opcode::RotateLeft | Opcode::RotateRight => {
+                Opcode::RotateLeft | Opcode::RotateRight | Opcode::SetFunctionName => {
```

Comment:
Same here.

Maybe making it a static method of `SetFunctionName` struct to  avoid code duplication.